content/reference/admin/api_tokens/index.md

```diff
@@ -9,24 +9,25 @@ weight: 20030
 
 ## Managing API Tokens
 
-To access the API Tokens administration, click on the {{< icon key >}} icon in the navigation bar.
```

Comment:
Why did you remove this line?

Reply:
We don't have the needed icons anymore. And I don't see the point of saying "To access API Tokens, click on API tokens"...

Reply:
You should at least say where the user needs to click to open the page, even if you don't display the icon.